R/SOS-methods-accessor.R

```diff
@@ -979,6 +979,7 @@ setMethod(f = "sosId", signature = signature(obj = "list"),
             return(sapply(obj, sosId))
           })
 
+# TODO Why not specified in "Generic-Methods.R"?
```

Comment:
Please remove comment.

Answer: Because the generic method is an accessor method... it grew this way. Added note in `DEV-README`.

Reply:
```suggestion
```

Reply:
@EHJ-52n Can you please commit this change? Thanks!